<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JVQL470" failures="1" tests="2" name="Test(failed)" time="44.491" errors="0" timestamp="2024-04-01T01:54:54 IST">
  <testcase classname="testCases.TC_002_LoginTest" name="verify_login" time="3.124"/>
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="verify_account_registration" time="5.369">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_001_AccountRegistrationTest.verify_account_registration(TC_001_AccountRegistrationTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_registration -->
</testsuite> <!-- Test(failed) -->
